src/wallet/rpcwallet.cpp

```diff
@@ -3515,6 +3516,207 @@ UniValue rescanblockchain(const JSONRPCRequest& request)
     return response;
 }
 
+class DescribeWalletAddressVisitor : public boost::static_visitor<UniValue>
+{
+public:
+    CWallet * const pwallet;
```

Comment:
Nit, m_wallet, why public?

Reply:
why not?

Reply:
See `CAffectedKeysVisitor`, `DestinationEncoder` and `CScriptVisitor` with private members. On the other hand `Witnessifier` members are public (could be "fixed") .

IMO all members, by default, should be private. When reading the code it's clear that a private member is not externally accessed/changed.